Source/Blazorise/Components/Modal/Modal.razor.cs

```diff
@@ -456,6 +467,11 @@ public Task EndAnimation( bool visible )
         /// </summary>
         protected internal bool IsVisible => State.Visible == true;
 
+        /// <summary>
+        /// Returns true if the modal backdrop should be visible.
+        /// </summary>
+        protected internal bool IsBackdropVisible = false;
```

Comment:
Rename to `BackdropVisible`. An `Is` prefix is usually used for property getters. Will be in the guidelines soon.